### ISSUE TITLE
E6. temporal extents for images and figures

### ISSUE THREAD

**Issue by zednis** [USGCRP/gcis-ontology#10]
Consider expanding/modifying gcis:startedAtTime and gcis:endedAtTime to include the temporal extents of figure and image content.  Currently, the domain of these terms is gcis:TemporalExtent which is solely a domain of a gcis:Dataset.  Maybe relate to dcterms:coverage or dcterms:temporal.

**Comment by zednis** [comment]
A few questions:
1) The gcis ontology does not appear to have a startedAtTime or endedAtTime property.
2). I do not understand the sentence 'Currently, the domain of these terms is gcis:TemporalExtent which is solely a domain of a gcis:Dataset.'
3) Could you provide an example?

I think that irregardless of the answers to my questions above we will want to use dcterms:coverage to describe the temporal extents of figure or image content.

Here is a guide on how to use the property correctly:
http://wiki.dublincore.org/index.php/User_Guide/Publishing_Metadata#dcterms:coverage

```
  <http://data.globalchange.gov/image/048de6b5-60ab-4ad1-895b-d41ad9bdfab5>
  a gcis:Image ;
  dcterms:coverage [ 
    a gcis:TemporalExtents ;
    gcis:startedAt "2015-05-24T00:00:00"^^xsd:dateTime ;
    gcis:endedAt "2015-05-25T00:00:00"^^xsd:dateTime ;
  ] ;
```

note - I did not use gcis:hasTemporalExtents here because
1) I think we should prefer usage of the dublin core property over the one we minted
2) gcis:hasTemporalExtents has a domain of gcis:ModelRun, which does not match this scenario

**Comment by justgo129** [comment]
It should be gcis:startedAt and gcis:endedAt, both of which have domains gcis:TemporalExtents and ranges xsd:dateTime.  Regarding questions 2 and 3 from @zednis, I wonder if I had used an earlier version of the ontology for this one.  Regardless, could we use gcis:startedAt and gcis:EndedAt for figures and images, or is dcterms:coverage really a preferred method (just asking as a point of information, not questioning your suggestion)?

**Comment by zednis** [comment]
You use both.  you use dcterms:coverage to link the image or figure to an extent and you use gcis:startedAt and gcis:endedAt to describe the extent.

**Comment by justgo129** [comment]
Are we all right to make this change?

**Comment by justgo129** [comment]
Closed #10 via https://github.com/USGCRP/gcis/pull/197
